src/main/java/com/greghaskins/spectrum/model/Hooks.java

```diff
@@ -75,29 +71,33 @@ public void runAround(final Description description, final RunNotifier notifier,
   }
 
   private void runAroundInternal(final Description description, final RunNotifier notifier,
-      final Block block) {
+      final Block block) throws Throwable {
     Variable<Boolean> hooksRememberedToRunTheInner = new Variable<>(false);
-    ThrowingConsumer<Block> consumer = innerBlock -> {
-      hooksRememberedToRunTheInner.set(true);
-      innerBlock.run();
-    };
+
+    Hook chainOfResponsibility = innerHook(hooksRememberedToRunTheInner);
 
     for (HookContext context : this) {
-      consumer = wrap(description, notifier, consumer, context);
+      chainOfResponsibility = wrap(chainOfResponsibility, context);
     }
-    consumer.accept(block);
+    chainOfResponsibility.accept(description, notifier, block);
 
     if (!hooksRememberedToRunTheInner.get()) {
       throw new RuntimeException("At least one of the test hooks did not run the test block.");
     }
   }
 
-  private ThrowingConsumer<Block> wrap(final Description description, final RunNotifier notifier,
-      final ThrowingConsumer<Block> inner, final HookContext outer) {
-    return block -> outer.getHook().acceptOrThrow(
-        wrapWithReporting(description, notifier, () -> inner.acceptOrThrow(block)));
+  private Hook innerHook(Variable<Boolean> hooksRememberedToRunTheInner) {
+    return (description, notifier, innerBlock) -> {
+      hooksRememberedToRunTheInner.set(true);
+      innerBlock.run();
+    };
   }
 
+  private Hook wrap(final Hook inner, final HookContext outer) {
+    return (description, notifier, block) -> outer.getHook().accept(description, notifier,
+        wrapWithReporting(description, notifier,
+            () -> inner.accept(description, notifier, block)));
```

Comment:
Moving away from the original `ThrowingConsumer` to this has enabled the reporting to see the real exceptions.